docs/automation/README.md

```diff
@@ -23,15 +23,37 @@ We expect your application will be managing endpoints to generate and accept a n
 
 An example of this is below:
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+...
+  #to request password reset
+  res = client.request(
+          method='POST',
+          data={"email":"user@email"},
+          url='/ums/resetpassword/'
+        ) 
+  #to apply changes
+  res = client.request(
+          method='POST',
+          data={"new_password": "jexia_super"},
+          url='ums/resetpassword/token' #token - user will get by email if you have Integration for SMTP
+        ) 
+  print(res)
+  
```

Comment:
pep8 again, and line 41 longer than 80 symbols (all IDEs will mark this line is incorrect)

Reply:
fixed